### DISCUSSION TITLE
Kibana doesn't show source IPs with a Count value less than 5

### DISCUSSION THREAD

**Discussion by OleksandrBolshov** [Security-Onion-Solutions/securityonion#6408] [General] [answered]
But in Hunt I see them. Those IPs are important and could be lost by looking only to Kibana. How to ensure that all information from PCAP is available in Kibana? Thanks (SO distributed v2.3.61)

**Comment by dougburks** [comment, marked as answer]
I suspect this may be due to the configuration of whatever visualization you're looking at as it may be limiting its result set. On the visualization, select the `Inspect` option and then go to `Request`. You'll most likely see something like `"size": 100` which would limit it to showing the top 100 IPs.

**Reply by OleksandrBolshov** [reply]
Exactly the value of the size is 100. Thanks
